lib/Common/Memory/HeapInfo.cpp

```diff
@@ -881,6 +881,7 @@ HeapInfo::AddLargeHeapBlock(size_t size)
 void HeapInfo::SweepBuckets(RecyclerSweep& recyclerSweep, bool concurrent)
 {
     Recycler * recycler = recyclerSweep.GetRecycler();
+    static_cast<Recycler*>(recycler);
```

Comment:
The same as above, a temporary workaround for unreferenced local variable.

---
In reply to: [66281536](https://github.com/Microsoft/ChakraCore/pull/1096#discussion_r66281536) [](ancestors = 66281536)